ddtrace/tracer.py

```diff
@@ -187,16 +201,13 @@ def configure(self, enabled=None, hostname=None, port=None, uds_path=None, https
         if isinstance(self.sampler, DatadogSampler):
             self.sampler._priority_sampler = self.priority_sampler
 
-        self._dogstatsd_host = dogstatsd_host or self._dogstatsd_host
-        self._dogstatsd_port = dogstatsd_port or self._dogstatsd_port
-        self.log.debug('Connecting to DogStatsd on {}:{}'.format(
-            self._dogstatsd_host,
-            self._dogstatsd_port,
-        ))
-        self._dogstatsd_client = DogStatsd(
-            host=self._dogstatsd_host,
-            port=self._dogstatsd_port,
-        )
+        if dogstatsd_host is not None and dogstatsd_port is not None and dogstatsd_url is None:
+            dogstatsd_url = '{}:{}'.format(dogstatsd_host, dogstatsd_port)
```

Comment:
```python
log.warning('deprecated: please use dogstatsd_url="%s:%d" instead' % (dogstatsd_host, dogstatsd_port))
```

Reply:
(or a real deprecation warning is fine too, not just a log)

Reply:
👍